provisioning/miniprov/hare_mp/main.py

```diff
@@ -288,18 +287,31 @@ def _start_hax(utils: Utils,
 
 
 def post_install(args):
+    try:
+        if checkRpm('cortx-motr') != 0:
+            logging.error('\'cortx-motr\' is not installed')
+            exit(-1)
+        if checkRpm('consul') != 0:
+            logging.error('\'consul\' is not installed')
+            exit(-1)
+        if checkRpm('cortx-hare') != 0:
+            logging.error('\'cortx-hare\' is not installed')
+            exit(-1)
```

Comment:
It seems like you're reverting some important changes in your commit. 
Can you make your changes minimal? I believe this reverting is not required by your PR (and even if it is, then such a change deserves a separate commit).

Reply:
Yes this was rebase related issue.

Resolved!